public/index.html

```diff
@@ -45,13 +40,14 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
 
-    <!-- The core Firebase JS SDK is always required and must be listed first -->
-    <script src="/__/firebase/7.4.0/firebase-app.js"></script>
+  <!-- The core Firebase JS SDK is always required and must be listed first -->
+  <!-- <script src="/__/firebase/7.4.0/firebase-app.js"></script> -->
```

Comment:
The only changes in this file are to comment out the firebase scripts. Rest is prettier